engine/spec/features/text_input_spec.rb

```diff
@@ -20,6 +20,12 @@
     end
   end
 
+  it "renders a page heading" do
+    within "#page_heading_text_field" do
+      expect(page).to have_css("h1.cads-page-title", text: "First name")
+    end
+  end
```

Comment:
I think I want to isolate these a little bit more and assert more e.g. a field with a page title doesn't support "optional" labels (as it doesn't make sense to have a single, but optional field. Also a little unclear how to debug when the test fails i.e. is it possible to _mount_ the test app (probably yes)